ddtrace/contrib/internal/graphql/patch.py

```diff
@@ -302,6 +304,31 @@ def _set_span_errors(errors: List[GraphQLError], span: Span) -> None:
     # we will not set the error.stack tag on graphql spans. Setting only one traceback
     # could be misleading and might obfuscate errors.
     span.set_tag_str(ERROR_MSG, error_msgs)
+    for error in errors:
+        locations = " ".join(f"{loc.formatted['line']}:{loc.formatted['column']}" for loc in error.locations)
+        attributes = {
+            "message": error.message,
+            "type": span.get_tag("error.type"),
+            "locations": locations,
+        }
+
+        if error.__traceback__:
+            stacktrace = "".join(
+                traceback.format_exception(
```

Comment:
I think `traceback.format_exc` might do what you want here more succinctly and without the need to `join`. If not, you might consider joining with `\n` for readability.